### PR TITLE
`station metrics`. Closes #23

### LINKED ISSUE #23: `station metrics`

### PR DESCRIPTION
**Pull request by juliangruber**
Closes #23

### CODE REVIEW

File: commands/metrics.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import fs from 'node:fs/promises'
+import { Tail } from 'tail'
+import { paths } from '../lib/paths.js'
+
+const metricsLogLineToJSON = (metrics) =>
+  JSON.stringify(
+    JSON.parse(metrics.split(/\[([^\]]+)\] /).pop()),
+    0,
+    2
+  )
```
Comment (juliangruber) [inline commands/metrics.js:10]: I'm currently not sure about this part. We're storing metrics as log lines prefixed by `[$date] `, and now parsing them in order to output them as JSON. If we say that `$ station metrics` will be the primary way to look at metrics, then I'd propose to use NLDJSON, just as pino uses. While not as convenient when inspecting the files manually, it is easier to integrate and work with. Wdyt @bajtos?

File: bin/station.js [hunk 2]
```diff
@@ -1,36 +1,15 @@
...
-  storagePath: join(paths.moduleStorage, 'saturn-L2-node'),
-  binariesPath: paths.moduleBinaries,
-  metricsStream: createLogStream(paths.metrics),
-  logStream: createLogStream(join(paths.moduleLogs, 'saturn-L2-node.log'))
-})
+if (process.argv.includes('metrics')) {
+  await commands.metrics()
+} else {
+  await commands.station()
+}
```
Comment (bajtos) [inline bin/station.js:15]: Have you considered using a library to parse CLI args? Node.js is adding built-in support, but it's experimental, so we should probably use something else like minimise or yargs. 

https://oclif.io seems to be a more modern option.

**Review by bajtos — APPROVED** [review]
Nice! Kudos for adding automated tests 👏🏻

File: bin/station.js (hunk 2 above)
Comment (juliangruber) [reply bin/station.js:15]: I have thought about it, but for now that would be overkill imo. The simple logic here works and doesn't need (a tree of) dependencies. I'd suggest to add a lib like this (would be happy to try oclif, or the trusted minimist/yargs) once we see the current approach doesn't scale or causes bugs

File: commands/metrics.js (hunk 1 above)
Comment (juliangruber) [reply commands/metrics.js]: I agree 👍  I've opened a new issue here for final discussion: https://github.com/filecoin-station/core/issues/37

I believe we can ship this PR without it for now

**Pull request merged**